platform/ui/src/components/ViewportActionCorners/ViewportActionCorners.tsx

```diff
@@ -20,19 +20,19 @@ const commonClasses = 'pointer-events-auto flex items-center gap-1';
 const classes = {
   [ViewportActionCornersLocations.topLeft]: classNames(
```

Comment:
fix action icons positing, and remove bottom spacing entirely as instructed by Dan